lib/iris/tests/results/unit/analysis/cartography/project/TestAll/cube.data.0.json

```diff
@@ -1 +1 @@
-{"std": 0.22496461010978716, "min": 287.27520751953125, "max": 288.09292602539062, "shape": [2, 3, 10, 10], "masked": true, "mean": 287.67258522727275}
\ No newline at end of file
+{"std": 0.22496461010978716, "min": 287.27520751953125, "max": 288.0929260253906, "shape": [2, 3, 10, 10], "masked": true, "mean": 287.67258522727275}
```

Comment:
Change is due to regeneration of target results, which has removed 14th decimal place (likely due to changes with numpy versions etcetera since the last time this was generated). Change in value is within an acceptable tolerance 👍